exonum-java-binding/core/rust/src/testkit/mod.rs

```diff
@@ -94,16 +106,17 @@ pub extern "system" fn Java_com_exonum_binding_testkit_TestKit_nativeCreateBlock
     handle: Handle,
 ) -> jbyteArray {
     let res = panic::catch_unwind(|| {
-        let testkit = cast_handle::<Box<TestKit>>(handle);
+        let testkit = cast_handle::<TestKit>(handle);
         let block = testkit.create_block().header;
-        let serialized_block = block.encode().unwrap();
-        let byte_array = env.byte_array_from_slice(&serialized_block)?;
-        Ok(byte_array)
+        let serialized_block = serialize_block(&env, block);
+        serialized_block
     });
     unwrap_exc_or(&env, res, std::ptr::null_mut())
 }
 
 /// Creates Block with specified list of transactions and returns its header.
+/// The transactions are the byte[][] array which contains the set of serialized transaction
```

Comment:
Then it cannot be a JList, it must be jobjectArray

Reply:
If a list is easier to use — then the Java signature must be changed accordingly (and this documentation)

Reply:
I thought array actually may serve as List. Fixed to jobjectArray